README.md

```diff
@@ -10,19 +10,25 @@
 This project provides a export library for XML and CSV generation according to the FINDOLOGIC export patterns.
 * XML <https://docs.findologic.com/doku.php?id=export_patterns:xml>
 * CSV <https://docs.findologic.com/doku.php?id=export_patterns:csv>
-  * Note that CSV support is experimental and not intended for production use yet.
+  * Note that CSV support is still relatively new. Consider it beta-quality.
 
 #### Export recommendation
 
 Using the XML export is recommended by FINDOLOGIC. The XML is easier to read and has some advantages over the CSV export like:
 
-* No encoding issues as the encoding attribute is provided in the XML response `<?xml version="1.0" encoding="UTF-8"?>`
-* Validation is more reliable
-* Simple escaping of content using the `<![CDATA[...]]>`-tag
-* Standardized structure
-* Dynamically extract the products from the database via `start` and `count` parameter in the url
-* No limited file size for XML because of pagination
-* Using multiple usergroups per product
+* No encoding issues as the encoding attribute is provided in the XML response `<?xml version="1.0" encoding="UTF-8"?>`.
+* Validation is more reliable.
+* Simple escaping of content using the `<![CDATA[...]]>`-tag.
+* Standardized structure.
+* Dynamically extract the products from the database via `start` and `count` parameter in the url.
+* No limited file size for XML because of pagination.
+* Using multiple usergroups per product.
+
+The key advantage for CSV is that it is possible to use way more groups and XML usergroups. On the other hand:
```

Comment:
should `use way more groups and XML usergroups`  be `use way more groups than XML usergroups`?

Reply:
:heavy_check_mark: